karma.conf.js

```diff
@@ -166,24 +169,14 @@ module.exports = function(config) {
 
     // level of logging
     // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
-    logLevel: config.LOG_INFO,
+    logLevel: process.env.ABLY_LOG_LEVEL,
```

Comment:
@SimonWoolf That's why it doesn't work :) 
Goal was to keep the Travis log shorter. Will set to `WARN` to see what happens.

Reply:
karma INFO-level logs add around 10 lines to the beginning of the test; the remaining ~40k lines are mostly from ably-js Logger. I'd focus on the 40k rather than the 10 lines to 5 ☺. Check out the `envPreprocessor` config line for exposing external env vars to karma tests

Reply:
Oh so [this](https://github.com/ably/ably-js/blob/add-browserstack-testing/.travis.yml#L11) doesn't work because `ABLY_LOG_LEVEL` doesn't appear in the `envPreprocessor` "enumeration"?

Reply:
Answering myself: yes :)